lib/routes.js

```diff
@@ -71,13 +71,13 @@ const setupRoutes = function setupRoutes() {
         }
       };
 
-      socket.invalid = (filePath = '<unknown>', compiler) => {
+      socket.invalid = (filePath, compiler) => {
```

Comment:
```suggestion
      socket.invalid = (filePath = '<unknown>', compiler) => {
```

I much prefer the default parameter syntax in this case. We can still accomplish the same outcome, however.